### PR TITLE
2 semi-layers of activation added for each batch

### PR DESCRIPTION
**Pull request by rajasekharmekala**

### CODE REVIEW

**Comment by rajasekharmekala** [comment]
results of Batch normalization  show significant decrement in standard deviation.

File: inspect_activations/TwoActivationFunctions/ann_two_activations_main.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    File=open(fileName,'a+')
+    File.write('\n\n\n')
+    File.write('## Iteration: ({:.0f}%)\n\n'.format(count))
+    File.write('| Layer 1\t\t| Layer 2\t\t| Layer 3\t\t| Accuracy\t\t|\n')
+    File.write('| ------------- |:-------------:|:-----:| -----:|\n')
+    File.close()
+    a_list=[F.relu, F.tanh, F.sigmoid, F.tanh]
+    c_list=[F.relu, F.tanh, F.sigmoid, F.tanh]
+    e_list=[F.relu, F.tanh, F.sigmoid, F.tanh]
+    for a in [F.sigmoid]:
```
Comment (nishnik) [inline inspect_activations/TwoActivationFunctions/ann_two_activations_main.py]: @rajasekharmekala I have suspicions on this line, please check!

File: inspect_activations/TwoActivationFunctions/ann_two_activations_main.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        if(a==F.relu):
+            b_list=[F.relu,F.tanh, F.sigmoid, F.selu]
+        elif(a==F.tanh):
+            b_list=[F.tanh,F.sigmoid, F.selu]
+        elif(a==F.sigmoid):
+            b_list=[F.sigmoid,F.selu]
+        else:
+            b_list=[F.selu]
+        for b in [F.sigmoid]:
```
Comment (nishnik) [inline inspect_activations/TwoActivationFunctions/ann_two_activations_main.py]: And this

File: inspect_activations/TwoActivationFunctions/ann_two_activations_main.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            for c in [F.sigmoid]:
```
Comment (nishnik) [inline inspect_activations/TwoActivationFunctions/ann_two_activations_main.py]: Similarly this

File: inspect_activations/TwoActivationFunctions/ann_two_activations_main.py [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+                if(c==F.relu):
+                    d_list=[F.relu,F.tanh, F.sigmoid, F.selu]
+                elif(c==F.tanh):
+                    d_list=[F.tanh,F.sigmoid, F.selu]
+                elif(c==F.sigmoid):
+                    d_list=[F.sigmoid,F.selu]
+                else:
+                    d_list=[F.selu]
+                for d in [F.sigmoid]:
```
Comment (nishnik) [inline inspect_activations/TwoActivationFunctions/ann_two_activations_main.py]: And this!

File: inspect_activations/batchNorm/main.py [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+x=0
+acc_arr=np.zeros((64,10))
+for count in range(0,10):
+    for a in [F.relu, F.tanh, F.sigmoid, F.selu]:
+        for b in [F.relu, F.tanh, F.sigmoid, F.selu]:
+            for c in [F.relu, F.tanh, F.sigmoid, F.selu]:
+                x+=1
+                print(x)
+                print(a, b, c)
+                solve(F.selu, F.selu,F.selu,0)
```
Comment (nishnik) [inline inspect_activations/batchNorm/main.py]: Hey, why are these necessary?
Won't it be `solve(a, b, c, count)`?

**Comment by rajasekharmekala** [comment]
They were broken computations.Thanks for pointing out.

**Comment by rajasekharmekala** [comment]
They were broken computations.Thanks for pointing out.
